core/src/main/java/com/flipkart/gjex/core/task/RotationManagementBasedHealthCheck.java

```diff
@@ -0,0 +1,54 @@
+package com.flipkart.gjex.core.task;
```

Comment:
RotationManagementBasedHealthCheck does not belong to core/task package . Can you please create a healthcheck package under code and move healthcheckRegistry.java as well to it .

Reply:
moved